Resources/Prototypes/Entities/Objects/Devices/Circuitboards/computer.yml

```diff
@@ -150,31 +150,37 @@
   name: cargo shuttle console board
   description: A computer printed circuit board for a cargo shuttle console.
   components:
-    - type: ComputerBoard
-      prototype: ComputerShuttleCargo
-    - type: StealTarget
-      stealGroup: CargoShuttleConsoleCircuitboard
+  - type: Sprite
+    state: cpu_supply
+  - type: ComputerBoard
+    prototype: ComputerShuttleCargo
+  - type: StealTarget
+    stealGroup: CargoShuttleConsoleCircuitboard
 
 - type: entity
   parent: BaseComputerCircuitboard
   id: SurveillanceCameraMonitorCircuitboard
   name: surveillance camera monitor board
   description: A computer printed circuit board for a surveillance camera monitor.
   components:
-    - type: ComputerBoard
-      prototype: ComputerSurveillanceCameraMonitor
-    - type: Tag
-      tags:
-        - SurveillanceCameraMonitorCircuitboard
+  - type: Sprite
+    state: cpu_security
+  - type: ComputerBoard
+    prototype: ComputerSurveillanceCameraMonitor
+  - type: Tag
+    tags:
+      - SurveillanceCameraMonitorCircuitboard
```

Comment:
```suggestion
    tags:
    - SurveillanceCameraMonitorCircuitboard
```

Reply:
You got me, I compared other entries yet again, instead of style guide. I am learning though.

Reply:
no worries, easy to miss when it is done differently everywhere